### DISCUSSION TITLE
Cell lost after performing LSI with lowering varFeatures

### DISCUSSION THREAD

**Discussion by Xiaxhan** [GreenleafLab/ArchR#1354] [Questions / Documentation] [answered]
Hi,

ArchR is a robust method and friendly to use! 
I also faced a problem and I really appreciated for your help.

As recommend by ArchR manual, I tried to tweak varFeatures when performing LSI. But when I lowered varFeatures, the number of cells which projected to the final LSI decreased.  
<img width="713" alt="Screen Shot 2022-03-22 at 22 46 09" src="https://user-images.githubusercontent.com/102193236/159633329-d0a47de2-a4a8-4a3c-9df5-50ade46f7640.png">

As the picture shows, the total cell number is 688,633. (After QC and doublets removal)
The "tt" reduced dimension is created by running addIterativeLSI with the default 25,000 varFeatures and the projected cell number is correct.
But "ttt" is the result after lowering to 20,000 varFeatures and the projected cells lost 2.

It seems not much but when I tried to lower to 10,000 varFeatures I lost about 30 cells.
These cells are actually still in my ArchRproject but they are lost and can't project to the specific LSI dimension.

I also try with the provided Hematopoiesis dataset but it's ok with lowering varFeatures - the projected cell number is still the total cell number.

I don't know why my dataset will occur this problem. Is it possible that my dataset is too large?(700,000 vs 10,000). But why?
I am wondering whether the V matrix generated by SVD will miss some information if dataset is large and some landmark cells are sampled to created the reduced dimensions? I can't figure out the detailed process.

Maybe it's just a small fraction of cells but i am really confused with the result.

Thank you so much!
Xia

**Comment by rcorces** [comment, marked as answer]
Looks like you are running into the issue described here https://github.com/GreenleafLab/ArchR/discussions/1345

**Reply by Xiaxhan** [reply]
I see! That makes sense.  

Thank you!
